renovate.json

```diff
@@ -16,6 +16,9 @@
       ],
       "groupName": "AppEngine packages"
     },
+    {
```

Comment:
Do we need to limit to our packages? e.i. "com.google.cloud" or whatever?

Reply:
I wish, but there is com.google.api, com.google.api-client, as well as com.google.cloud.  (maybe a few more)